azure-proto-core/Resources/UserAssignedIdentity.cs

```diff
@@ -0,0 +1,118 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+namespace azure_proto_core.Resources
+{
+    using System;
```

Comment:
Move these outside the namespace, it was an old stylecop thing that was asking for these to be inside